core/object/script_language.h

```diff
@@ -386,6 +386,7 @@ class ScriptLanguage : public Object {
 	virtual String debug_get_stack_level_function(int p_level) const = 0;
 	virtual String debug_get_stack_level_source(int p_level) const = 0;
 	virtual void debug_get_stack_level_locals(int p_level, List<String> *p_locals, List<Variant> *p_values, int p_max_subitems = -1, int p_max_depth = -1) = 0;
+	virtual void debug_get_stack_level_locals(int p_level, Vector<String> *p_locals, Array *p_values, int p_max_subitems = -1, int p_max_depth = -1) = 0;
```

Comment:
Can't we merge those into a single method, and convert the input datatype in one of the method call sites? It feels strange to have an overload for a method with an equivalent type.

Reply:
I overloaded it for not adding an extra load with converting data.. I don't know that if Godot types (`Array` and `List<T>`) could be casted/reinterpreted as `Vector<T>` and `Array`. If it will add an extra load to runtime, I think an overloaded version is better imo.